arelleCmdLine.py

```diff
@@ -6,12 +6,14 @@
 '''
 import os
 import sys
-import regex as re
 
+# Plugin imports above this line will result in module not found exceptions on the frozen macOS build.
```

Comment:
Out of curiosity, what does "plugin import" mean?  Why is regex considered a "plugin"?

Reply:
It's not. I meant library imports. Don't write documentation right after context switching 🤦‍♂️

Reply:
Heh, my Python knowledge is weak, so I'm not sure what "library import" means either.  Would "Installed package imports above this line" be more accurate?